src/lib/checkbox/checkbox.ts

```diff
@@ -164,6 +164,13 @@ export class MdCheckbox implements ControlValueAccessor, AfterViewInit, OnDestro
 
   @ViewChild(MdRipple) _ripple: MdRipple;
 
+  @ViewChild('labelWrapper') _labelWrapper: ElementRef;
+
+  get hasLabel(): boolean {
```

Comment:
Let's change this to a method rather than a getter. It should also be marked as internal (name with a leading underscore). It also needs a JsDoc description comment.